wifiutils/src/main/java/com/thanosfisherman/wifiutils/ConfigSecurities.java

```diff
@@ -111,7 +111,7 @@ static void setupSecurity(@NonNull WifiConfiguration config, String security, @N
     }
 
     @TargetApi(Build.VERSION_CODES.Q)
-    static void setupWifiNetworkSpecifierSecurities(@NonNull WifiNetworkSpecifier.Builder wifiNetworkSpecifierBuilder, String security, @NonNull final String password) {
+    public static void setupWifiNetworkSpecifierSecurities(@NonNull WifiNetworkSpecifier.Builder wifiNetworkSpecifierBuilder, String security, @NonNull final String password) {
```

Comment:
Is there any particular reason why we should expose all these methods in `ConfigSecurities` as public?